src/main/java/rps/CommandLinePrompt.java

```diff
@@ -47,8 +48,23 @@ public ReplayOption readValidReplayOption() {
         return ReplayOption.of(readInput());
     }
 
+    @Override
+    public void displayChosenMove(Gesture gesture, String name) {
+        write(name + " chose " + gesture.getId() + " - " + gesture.name());
+    }
+
+    @Override
+    public void displayWinner(String winnerName) {
+        write(winnerName + " won");
+    }
+
+    @Override
+    public void displayDraw() {
+        write("Draw");
+    }
+
     private String formatGesturesForPrompt() {
-        Gesture[] gestures = Gesture.values();
+        Gesture[] gestures = values();
```

Comment:
I am not sure I do like the name of this method `values()`. If I'm thinking about the context of what I do understand (without necessarily looking) a `CommandLinePrompt` does, I don't seem to understand what `values()` means.

Is there a better name for this?